utilities/conv_gefs2datm/0readme

```diff
@@ -0,0 +1 @@
+conv_gefs2datm
```

Comment:
Can you add instructions on what to do or where to find more information?

Reply:
some imformation added to 0readme file

Reply:
Thanks! This is great information.  Can you also add information on how to run the conversion from the original file format to netcdf? And either list or say where we can find more information on if any assumptions were made or how values were calculated?

Reply:
Updated 0readme file

Reply:
Thank you for the instructions on how to run the script.  Can you also put where to find more information about any fields that are modified other than being changed to netcdf format?

Reply:
Thanks!